README.md

```diff
@@ -2,55 +2,54 @@
 
 # Project Name
 
-> Description the project.
-
+> Awesome Books
```

Comment:
- [ ] **Required:** Kindly remove the **Project Name** placeholder and make **Awesome Books** the title.